CiJenkinsfile

```diff
@@ -0,0 +1,193 @@
+import java.text.SimpleDateFormat
+
+
+pipeline {
+    agent {
+        kubernetes {
+            label "builder-pod-${UUID.randomUUID().toString()}"
+            defaultContainer 'jnlp'
+            yaml """
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    some-label: builder-pod
+spec:
+  containers:
+  - name: helm
+    image: vfarcic/helm:2.8.2
+    command:
+    - cat
+    tty: true
+  - name: golang
+    image: golang:1.9
+    command: ["cat"]
+    tty: true
+  - name: git
+    image: alpine/git
+    command:
+    - cat
+    tty: true
+  - name: git
+    image: digitalinside/gren:latest
+    command:
+    - cat
+    tty: true    
+  - name: kubectl
+    image: vfarcic/kubectl
+    command: ["cat"]
+    tty: true
+  - name: docker
+    image: docker
+    command:
+    - cat
+    tty: true
+    volumeMounts:
+    - mountPath: /var/run/docker.sock
+      name: docker-sock-volume
+  volumes:
+  - name: docker-sock-volume
+    hostPath:
+      path: /var/run/docker.sock
+      type: File
+"""
+        }
+    }
+
+    parameters {
+        string(name: 'project', defaultValue: 'go-demo-3')
+        string(name: 'image', defaultValue: 'digitalinside/go-demo-3')
+        string(name: 'domain', defaultValue: '192.168.0.10.nip.io')
+        string(name: 'addr', defaultValue: 'go-demo-3.192.168.0.10.nip.io')
+        string(name: 'cmAddr', defaultValue: 'cm.4.192.168.0.10.nip.io')
+        string(name: 'chartVer', defaultValue: '0.0.1')
+    }
+
+    options {
+        timeout(time: 45, unit: 'MINUTES')
+    }
+
+    stages {
+        stage("prep") {
+            steps {
+                script {
```

Comment:
Move to a library? Once the fix is done, we can just change the implementation of the library.